### ISSUE TITLE
insider reflow validation: Overview percent text fails color contrast when selected

### ISSUE THREAD

**Issue by pownkel** [microsoft/accessibility-insights-web#3078]
## Describe the bug

The dark grey used for the Overview percent completed text does not have sufficient contrast against the nav selection color:

<img width="177" alt="overview percent" src="https://user-images.githubusercontent.com/55459788/87479162-d7601a80-c5df-11ea-85fa-7768d3ca3d2c.PNG">

Contrast should be at least 4.5:1 for regular text.

## Context (please complete the following information)

- AI-Web Version & Environment: 2.19.0 Insider
